ui/src/containers/preferences.list/details.tsx

```diff
@@ -0,0 +1,206 @@
+import PageHeader from "~/components/page-header";
+import {
+  Button,
+  Flex,
+  Grid,
+  Separator,
+  Switch,
+  Text,
+  TextField,
+} from "@raystack/apsara";
+import { useCallback, useEffect, useState } from "react";
+import { V1Beta1Preference, V1Beta1PreferenceTrait } from "@raystack/frontier";
+import { useParams } from "react-router-dom";
+import { useFrontier } from "@raystack/frontier/react";
+import Skeleton from "react-loading-skeleton";
+import dayjs from "dayjs";
+import * as R from "ramda";
+import { toast } from "sonner";
+
+interface PreferenceValueProps {
+  trait: V1Beta1PreferenceTrait;
+  value: string;
+  onChange: (v: string) => void;
+}
+
+function PreferenceValue({ value, trait, onChange }: PreferenceValueProps) {
+  if (R.has("checkbox")(trait)) {
+    const checked = value === "true";
+    return (
+      <Switch
+        // @ts-ignore
+        checked={checked}
+        onCheckedChange={(v: boolean) => onChange(v.toString())}
+      />
+    );
+  } else if (R.or(R.has("text"), R.has("textarea"))(trait)) {
+    return (
+      <TextField value={value} onChange={(e) => onChange(e.target.value)} />
+    );
+  } else {
```

Comment:
Not able to edit 
- `Invite Mail Body`
- `Full name`
etc. 


Maybe they have different type. Can you look into that






<br class="Apple-interchange-newline">

Reply:
`Full name` is fixed, there is no field type for `Invite Mail Body`. we will fix that in backend.